runusb

```diff
@@ -136,6 +194,19 @@ class RobotUSBHandler(USBHandler):
             return
         os.killpg(self.process.pid, sig)
 
+    def _watch_process(self) -> None:
+        # Wait for the process to complete
+        self.process.wait()
```

Comment:
Advise keeping a sleep(1) between the robot code exiting and _set_leds being called; otherwise, if the robot code exits immediately (e.g. syntax error) you won't see the LED go yellow before going back to red. Here is probably a good place for that.

Reply:
I'd intentionally left it out, but on reflection because the "I have no code" and "This code crashed" states are the same, there's no way to identify exactly what's going on.

I think a review of the states we have and the LEDs we have available would be good, but for now maintaining existing functionality is probably the best way forward.